Makefile

```diff
@@ -49,7 +49,7 @@ setup: ## Setup local environment for development, generate configuration files.
 init: ## Initialize your local environment and start the core solution.
 	$(info Initialize your local environment and start the core solution)
 	@make setup
-	@make up
+	@make up p=init
```

Comment:
Default only run the basic services that need to be initialized (i.e. database, kafka, elasticsearch, and keycloak)